docs/section11/Rule_11-7.md

```diff
@@ -0,0 +1,47 @@
+# Service_Water_Heating - Rule 11-7  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-7  
+
+**Rule Description:** Except in buildings that will have no service water heating loads,  the service water heating systems type in the baseline building design shall be as specified in Table G3.1.1-2 for each building area type in the proposed design. 
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a & b
+
+**Evaluation Context:** B-RMD each SWH Equipment
+**Data Lookup:**   
+- **Table G3.1.1-2(shw_bat)**
+**Function Call:** 
+- **get_component_by_id**
+- **get_SHW_equipment_associated_with_each_SWH_bat**  
+- **get_SHW_types_and_SHW_use**
+- **get_SHW_equipment_type**  
```

Comment:
I think some of these function names have been revised. Recommend updating accordingly.